docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -85,7 +85,9 @@ ifdef::context[:parent-context: {context}]
 
 // Purpose statement for the assembly
 [role="_abstract"]
-As an owner of a Kafka instance in {product-long-kafka}, you can manage the level of access that other user accounts and service accounts have to your instance. You can allow or deny access to your instance for specific accounts or for all accounts in your organization. You can also allow other users or service accounts to manage the level of access to your instance for you.
+As an owner of a Kafka instance in {product-long-kafka}, you control access permissions for all resources within that Kafka instance by default. This includes topics, groups, transactional IDs, and Access Control Lists (ACLs) themselves. Other users cannot see permissions for your Kafka instance in the {product-long-kafka} web console or from Kafka APIs.
```

Comment:
The owner's permissions are "special" - they are inherent (cannot be changed) and they are don't appear in the permission list for either the owner or other users.

* "by default" - to me conveys an initial state and leaves open the possibility that things can be changed.  That's not the case here.  The owner's permissions are set in stone.
* "Other users cannot see permissions for your Kafka instance in the {product-long-kafka} web console or from Kafka APIs." is unclear.     It is not just "other users" who don't see the owner's permission.  It is the owner too.   Also we are not talking about all permissions - just this special set of owner permisisons.




This is not conveying that 

this set of permissions held by the owner is immutable.
this set of permissions is implicit - that means to say that user won't see these permissions enumerated in the console or via the API.

Reply:
Thanks for the clarification @k-wall . Would something like this work (trying to keep the language as plain as possible for better readability):

You also control a set of permissions for all resources within a Kafka instance including topics, groups, transactional IDs, and Access Control Lists (ACLs). This set of permissions cannot be changed and cannot be seen by you or any other user in the **Permission** list in the {product-long-kafka} web console or from Kafka APIs.